.github/workflows/base_node_binaries.json

```diff
@@ -16,7 +16,11 @@
     "cross": true,
     "target_bins": "--bin tari_base_node --bin tari_console_wallet --bin tari_merge_mining_proxy --bin tari_miner",
     "target_cpu": "generic",
-    "features": "safe"
+    "features": "safe",
+    "networks": [
```

Comment:
This is only going to build multiple net versions for linux-arm64. I am thinking if we have different nets for different builds, it's worth having in the json file, but if we will always be building for the same nets, then best add as an include in the matrix for the workflow. I will send an example.